Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/MediaPlayerElementSamplePage.xaml

```diff
@@ -0,0 +1,214 @@
+﻿<Page x:Class="Uno.Gallery.Views.Samples.MediaPlayerElementSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  mc:Ignorable="d">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.FluentTemplate>
```

Comment:
we are hotlinking lots of media files here, thats ok?

Reply:
meaning?

Reply:
we are hotlinking a bunch of urls that dont belong to us
this could be a problem, as we dont knowthey will stay available for how long

Reply:
The Buck Bunny videos are fine, although we need to check to make sure those links are the correct source links for Buck Bunny - this is a standard set of video files that most streaming services use to test their players when in development (and they've been around for almost a decade).
The other videos I'm not sure but agreed for Gallery, we should be using self hosted, or videos from a known site (with authority to be used for app dev testing)

Reply:
URLs updated for self hosted uno videos